examples/logging_handler.py

```diff
@@ -0,0 +1,47 @@
+"""
+Show the usage of influx with python native logging.
+
+This is useful if you
+* want to log to influx and a local file.
+* want to set up influx logging in a project without specifying it in submodules
+"""
+import datetime
+import logging
+import time
+
+from influxdb_client import InfluxLoggingHandler, WritePrecision, Point
+
+DATA_LOGGER_NAME = '…'
+
+
+def setup_logger():
+    """
+    Set up data logger with the influx logging handler.
+
+    This can happen in your core module.
+    """
+    influx_logging_handler = InfluxLoggingHandler(url="…", token="…", org="…", bucket="…",
+                                                  client_args={'arg1': '…'},
+                                                  write_api_args={'arg': '…'})
```

Comment:
Please add an example of a valid values for `client_args` and `write_api_args`.

```suggestion
                                                  client_args={'timeout': 30_000},
                                                  write_api_args={'write_options': SYNCHRONOUS})
```